### ISSUE TITLE
Wrong version number in source files

### ISSUE THREAD

**Issue by GamingMinds-DanielC** [epezent/implot#498]
Small issue: line 23 in all source files and line 63 in `implot.h` still state version 0.14 in the 0.15 release.

**Comment by ocornut** [comment]
I have fixed those and added changelog entries with 85573fe.

I would like to update the tag/release now, which may involve republishing the release.

**Comment by ocornut** [comment]
Done.
https://github.com/epezent/implot/releases/tag/v0.15

Here's exactly was I did (after I exercised it on a dummy repo).


- I have remotely deleted v0.15 tag via git on my machine.
- At that point the v0.15 release don't show for users. But it shows as a draft for project mods.
- I could resubmit v0.15 with the new commit.

Interestingly, the updated v0.15 release still shows @epezent as author, and the previous emojis/likes preserved, which is technically all sort of undefined behaviors: it's not totally considered a new release from github pov. Hopefully won't throw anyone's off, only the literal version string + comments were changed.
